biblio/how_to_cite_gudhi.bib

```diff
@@ -2,141 +2,157 @@ @book{gudhi:urm
 , title        = "{GUDHI} User and Reference Manual"
 , author      = "{The GUDHI Project}"
 , publisher     = "{GUDHI Editorial Board}"
-, year         = 2015
-, url =    "http://gudhi.gforge.inria.fr/doc/latest/"
+, edition =     "{3.0.0}"
+, year         = 2019
+, url =    "https://gudhi.inria.fr/doc/3.0.0/"
 }
 
 @incollection{gudhi:FilteredComplexes
 , author =  "Cl\'ement Maria"
 , title =   "Filtered Complexes"
 , publisher =  "{GUDHI Editorial Board}"
+, edition =     "{3.0.0}"
 , booktitle =   "{GUDHI} User and Reference Manual"
-, url = "http://gudhi.gforge.inria.fr/doc/latest/group__simplex__tree.html"
-, year =        2015
+, url = "https://gudhi.inria.fr/doc/3.0.0/group__simplex__tree.html"
```

Comment:
Hmm, now we will have to remember to update this before each release (the next release isn't going to be 3.0.0)... I guess we could generate this file from cmake so the release is automatically inserted. Do you have other ideas?

Reply:
And what about the CGAL one ? Do we have to update it at each release or not ?
Your idea of bibtex generation is a really good idea. I will have a look at it.

Reply:
> And what about the CGAL one ? Do we have to update it at each release or not ?

Yeah, I don't know about that one, seems less important and I can't think of a very good solution. Maybe just update it once in a while, when we remember?

Reply:
It is now generated for GUDHI on e1404be9da9cda8e6f41e7cf207934f1914a3505